spec/factories/pipeline_runs.rb

```diff
@@ -4,18 +4,31 @@
       # Array of taxon_counts entries to create.
       # The hash elements will be passed on to taxon_count factory as keyword arguments.
       taxon_counts_data { [] }
+      amr_counts_data { [] }
+      output_states_data { [] }
       # Array of pipeline_run_stage entries to create.
       # The hash elements will be passed on to pipeline_run_stage factory as keyword arguments.
       pipeline_run_stages_data { [] }
     end
 
     alignment_config { create(:alignment_config) }
 
+    trait :with_amr_count do
```

Comment:
This can be removed since you are only using the `amr_counts_data` variable below.

Reply:
Removed!